i.sentinel_2.vrt.index/i.sentinel_2.vrt.index.py

```diff
@@ -0,0 +1,226 @@
+#!/usr/bin/env python3
+############################################################################
+#
+# MODULE:       i.sentinel_2.vrt.index
+# AUTHOR(S):    Guido Riembauer
+# PURPOSE:      Creates vrts from input rasters with identical Band suffix and
+#               calculates indices parallelly.
+# COPYRIGHT:    (C) 2020-2023 by mundialis GmbH & Co. KG and the
+#               GRASS Development Team
+#
+#  This program is free software; you can redistribute it and/or modify
+#  it under the terms of the GNU General Public License as published by
+#  the Free Software Foundation; either version 2 of the License, or
+#  (at your option) any later version.
+#
+#  This program is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#  GNU General Public License for more details.
+
+#
+############################################################################
+
+# %module
+# % description: Creates .vrt from input rasters and calculates different indices and texture measures in parallel.
```

Comment:
```suggestion
# PURPOSE:      Creates VRTs from input rasters with identical band suffix and
#               calculates indices in parallel.
# COPYRIGHT:    (C) 2020-2023 by mundialis GmbH & Co. KG and the
#               GRASS Development Team
#
#  This program is free software; you can redistribute it and/or modify
#  it under the terms of the GNU General Public License as published by
#  the Free Software Foundation; either version 2 of the License, or
#  (at your option) any later version.
#
#  This program is distributed in the hope that it will be useful,
#  but WITHOUT ANY WARRANTY; without even the implied warranty of
#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
#  GNU General Public License for more details.

#
############################################################################

# %module
# % description: Creates GRASS GIS VRT maps from input raster maps and calculates different indices and texture measures in parallel.
```